### PR TITLE
list track ui

### PR DESCRIPTION
**Pull request by victorbaxter**
## [Pull request title]
list track ui
### [Redmine link]
https://edu-redmine.sun-asterisk.vn/issues/7673
#### Screenshots (nếu có)
Screenshot_1552966487

### CODE REVIEW

**Comment by victorbaxter** [comment]
ready

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -12,6 +12,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.AppCompat.Light.NoActionBar">
+        <activity android:name=".ui.listtrack.ListTrackActivity"></activity>
```
Comment (NguyenMinhTuanB) [inline app/src/main/AndroidManifest.xml]: convention

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -26,4 +27,4 @@
         <service android:name=".service.PlayMusicService" />
     </application>
 
-</manifest>
+</manifest>
```
Comment (NguyenMinhTuanB) [inline app/src/main/AndroidManifest.xml]: need an endline

File: app/src/main/res/layout/item_list_track.xml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+    <android.support.constraint.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/color_white">
+
+        <ImageView
+            android:id="@+id/image_track"
+            android:layout_width="@dimen/dp_100"
+            android:layout_height="match_parent"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/item_list_track.xml]: not match parent

**Comment by victorbaxter** [comment]
ready

**Pull request merged**